docs/modeling/physical_models.rst

```diff
@@ -288,3 +288,38 @@ The maximum circular velocity and radius of maximum circular velocity are availa
     # Display plot
     plt.tight_layout(rect=[0, 0.03, 1, 0.95])
     plt.show()
+
+
+.. _Cosmologies:
+
+Cosmologies
+===========
+
+The instances of the |Cosmology| class (and subclasses) include
+|Cosmology.to_format|, a method to convert a Cosmology to another python
+object. Relevant here is the conversion of any redshift method to a
+:class:`~astropy.modeling.FittableModel` instance, using the argument
+``format="astropy.model"``.
+Each |Cosmology| :class:`~astropy.cosmology.Parameter` is converted to a
+:class:`astropy.modeling.Model` :class:`~astropy.modeling.Parameter`
+and the redshift-method to the model's ``__call__`` / ``evaluate``.
+Now you can fit cosmologies with data!
+
+.. code-block::
+
+    >>> from astropy.cosmology import Planck18
+    >>> model = Planck18.to_format("astropy.model", method="lookback_time")
```

Comment:
```suggestion
    >>> model = Planck18.to_format(format="astropy.model", method="lookback_time")
```
Add the keyword for consistency with proceeding paragraph and the conversion example below.

Reply:
👍